src/commonMain/kotlin/space/kscience/gdml/GdmlMaterialPostProcessor.kt

```diff
@@ -0,0 +1,72 @@
+package space.kscience.gdml
+
+public class GdmlMaterialPostProcessor {
```

Comment:
Make it `object GdmkMaterialPostProcessor` and move everything from companion to the object body.

Reply:
Even better make it `fun Gdml.removeUnusedMaterials()` extension.

Reply:
I have implemented the changes.